src/platform/Linux/BUILD.gn

```diff
@@ -123,6 +128,10 @@ static_library("Linux") {
   }
 
   if (chip_enable_wifi) {
+    sources += [
+      "GlibTypeDeleter.h",
```

Comment:
Should we move this to global sources? given that both thread and wifi need it.

Or if we do ethernet only this is not needed hence the separate logic?

Reply:
@andy31415 Would the need for this to be moved show up in any action workflow failure?

Reply:
I think any changes this could be done as followups. Closing conversation for now.